posts/azure-seen-by-developers.md

```diff
@@ -0,0 +1,190 @@
+
+![Figure 9 : Application Insights online documentation](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_9.png)
+
+To configure Application Insights (Azure Monitor feature) in your application, you can follow [this documentation](https://docs.microsoft.com/fr-fr/azure/azure-monitor/app/app-insights-overview) which redirects to the web page shown in Figure 9.
+
+## Databases	
+
+![Figure 10 : View of a SQL Server DB](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_10.png)
+
+This view is the home page of a SQL Server DB. Among other things, it allows you to access the SQL Server via the _srvint_ link located on the right side of the screen. It also provides two monitoring indicators on the DB which are the CPU usage and the disk fill rate.
+
+In the left menu there are useful shortcuts such as "Query Performance Insight" which informs about the performance of the database by type of metrics and "Connection Strings" which lists the connection strings to the DB.
+
+![Figure 11 : Query Performance Insight page view](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_11.png)
+
+This list view allows you to visualize the performance of your DB with several very useful filters. Indeed, you can choose to observe the CPU performance, but also the input/output in terms of data or logs, the duration of the queries and finally the number of execution of the queries, all on time intervals chosen by another filter. Thus, this tool can be very useful when the need to analyze or improve the performance of the DB is felt.
+
+Let's now move to the "Connection Strings" view.
+
+![Figure 12 : View of the Connection Strings page](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_12.png)
+
+This view lists the different connection strings available for C#, C++, Java, PHP, Go and finally the ODBC standard for more openness. Normally, you should see these strings in the boxes but they have been removed for confidentiality reasons. All you have to do is to copy the URL you are interested in into your own DB access configuration and customize your authentication information, the famous user/password!
+
+Now go back to the DB overview page to go to the DB server by clicking on the _srvint_ link.
+
+![Figure 13 : View of a SQL Server](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_13.png)
+
+From this screen, you have the contact of the DB administrator in the "Active Directory Admin" field in case you need support and a quick link back to _dbint_ that redirects you to the DB.
+
+In the left menu there are some useful shortcuts like SQL databases which again allows you to find the link to the DB or _Firewalls and virtual networks_ which allows you to manage the IP addresses allowed to connect to the DB.
```

Comment:
In the left menu, there are some useful shortcuts, such as SQL databases, which allow you to find the link to the DB or _Firewalls, and virtual networks_ which allows you to manage the IP addresses allowed to connect to the DB.